### ISSUE TITLE
What social media tags should tokens() preserve?

### ISSUE THREAD

**Issue by koheiw** [quanteda/quanteda#1923]
In developing #1922, I belatedly noticed that there are many different tags are used in social media platforms. I am no social media expert. Can anyone suggest more?

- Twitter: `#abc`, `@abc`
- Weibo: `#abc#`
- Google Plus: `+abc`, `@abc`

We also should not forget about non-social media.

- Some legal documents: `##abc`

**Comment by kbenoit** [comment]
A good candidate for a new `quanteda_options()` parameter: 

```r
tag_patterns = c("#\\w+", "#\\w+#", "+\\w")
username_patterns = c("@\\w+")
```

**Comment by koheiw** [comment]
Solved via #1922
